crates/admin/src/cluster_controller/mod.rs

```diff
@@ -10,6 +10,7 @@
 
 pub mod cluster_state_refresher;
 mod logs_controller;
+mod nodeset_selection;
```

Comment:
This could become a sub module of `logs_controller`.

Reply:
Done, does this look reasonable? I'm a bit uncertain if this is the right directory layout for the modules as they stand.